packages/terra-hyperlink/tests/wdio/hyperlink-spec.js

```diff
@@ -32,8 +32,11 @@ describe('Hyperlink', () => {
 
     Terra.should.beAccessible();
     Terra.should.matchScreenshot();
-    Terra.should.themeEachCustomProperty({
-      '--terra-hyperlink-audio-after-background-image': 'linear-gradient(red, orange)',
+    Terra.should.themeCombinationOfCustomProperties({
+      testName: 'themed',
```

Comment:
I'm noticing you keep adding the same theme name. It might be worth logging an issue to terra-toolkit to set the default name to be 'themed' for this helper

Reply:
I have logged an issue regarding it. https://github.com/cerner/terra-toolkit/issues/274